packages/nimble-components/src/table/components/row/template.ts

```diff
@@ -28,11 +28,12 @@ export const template = html<TableRow>`
     >
         ${when(x => !x.rowOperationGridCellHidden, html<TableRow>`
             <span role="gridcell" class="row-operations-container">
-                ${when(x => x.selectable && !x.hideSelection, html<TableRow>`
+                ${when(x => x.showSelectionCheckbox, html<TableRow>`
                     <${checkboxTag}
                         ${ref('selectionCheckbox')}
                         class="selection-checkbox"
-                        @change="${(x, c) => x.onSelectionChange(c.event as CustomEvent)}"
+                        tabindex="-1"
```

Comment:
Chatted offline and agreed that it's a little wierd for these to be statically defined in the template but then mutated out-of-band. We should add comments to these tabindex definitions that will be manipulated by the keyboard mananager and create a tech debt to see if we can make these bound to named properties that can be manipulated instead (i.e. row.selectionCheckboxTabIndex, etc)

Reply:
Added comments in last PR update (`tabindex managed dynamically by KeyboardNavigationManager`).

#2211